lib/rubocop/cop/betterment/authorization_in_controller.rb

```diff
@@ -34,10 +34,9 @@ class AuthorizationInController < Cop
         PATTERN
 
         def initialize(config = nil, options = nil)
-          super(config, options)
-          config = @config.for_cop(self)
-          @unsafe_parameters = config.fetch("unsafe_parameters", []).map(&:to_sym)
-          @unsafe_regex = Regexp.new config.fetch("unsafe_regex", ".*_id$")
+          super
+          @unsafe_parameters = cop_config.fetch("unsafe_parameters").map(&:to_sym)
```

Comment:
I replaced `@config.for_cop` with `cop_config` and RuboCop started complaining because these configurations should exist in `config/default.yml`.